js/api/locationObserver/index.js

```diff
@@ -0,0 +1,50 @@
+const ReactNative = require('react-native');
```

Comment:
Please follow the convention of other classes where the file name have the same case than the default class it exports, `LocationObserver` in this case.